dataland-frontend/tests/e2e/specs/sme/DataIntegrity.ts

```diff
@@ -0,0 +1,98 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { admin_name, admin_pw, getBaseUrl } from "@e2e/utils/Cypress";
+import { getKeycloakToken } from "@e2e/utils/Auth";
+import {
+  Configuration,
+  type DataMetaInformation,
+  DataTypeEnum,
+  SmeDataControllerApi,
+  type SmeData,
+  type StoredCompany,
+} from "@clients/backend";
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from "@e2e/utils/CompanyUpload";
+import { submitButton } from "@sharedUtils/components/SubmitButton";
+import { uploadFrameworkData } from "@e2e/utils/FrameworkUpload";
+import { compareObjectKeysAndValuesDeep } from "@e2e/utils/GeneralUtils";
+import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+
+let smeFixtureForTest: FixtureData<SmeData>;
+before(function () {
+  cy.fixture("CompanyInformationWithSmePreparedFixtures").then(function (jsonContent) {
+    const preparedFixturesSme = jsonContent as Array<FixtureData<SmeData>>;
+    smeFixtureForTest = getPreparedFixture("Sme-dataset-with-no-null-fields", preparedFixturesSme);
+  });
+});
+
+describeIf(
+  "As a user, I expect to be able to edit and submit Sme data via the upload form",
+  {
+    executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+  },
+  function (): void {
+    let tokenForAdminUser: string;
+    let storedTestCompany: StoredCompany;
+    let dataMetaInfoOfTestDataset: DataMetaInformation;
+    before(() => {
+      const uniqueCompanyMarker = Date.now().toString();
+      const testCompanyName = "Company-Created-In-Sme-Blanket-Test-" + uniqueCompanyMarker;
+      getKeycloakToken(admin_name, admin_pw)
```

Comment:
No need for two before blocks

Reply:
Habe alles in den einen before block auf oberster Ebene gepackt.